components/bl-rating-component/README.md

```diff
@@ -0,0 +1,31 @@
+# Rating
+Rating is a component of Backendless UI-Builder designer. It allows you to rate anything, or see the rating of a particular item.
+
+## Usage
+
+Add a component to your page and select the initial rating values.
+
+### Component Elements
+
+<dl>
+<dt>Icon</dt>
+<dd>Each icon has its own rating, when you click on it, the rating will change depending on the value of a particular icon.</dd>
+</dl>
+
+### Component Properties
+
+  Name              | Default value     | Description
+ -------------------|-------------------|-------------------------------------------------------------------------------------
+  Disabled          | false             | If you need the rating to be disabled, you can click on the `Disabled` checkbox.
+  Icon              | Star              | You can select the icon to be displayed.
```

Comment:
Default value is "star"(add quotes as it string), Star is label. Also for select property type would be helpful to write all possible values at description.